_overviews/collections/conversions-between-java-and-scala-collections.md

```diff
@@ -27,8 +27,8 @@ Sometimes you might need to pass from one collection framework to the other. For
 
 To enable these conversions, simply import them from the [JavaConverters](https://www.scala-lang.org/api/{{ site.scala-212-version }}/scala/collection/JavaConverters$.html) object:
 
-    scala> import collection.JavaConverters._
-    import collection.JavaConverters._
+    scala> import scala.jdk.CollectionConverters._
+    import scala.jdk.CollectionConverters._
```

Comment:
Only the overview for the scala 2.13 collections should be changed I think, the overview for the old collections should stay as is.

Reply:
gah, how'd I miss that, good catch